data/assets/scene/solarsystem/heliosphere/bastille_day/fieldlines.asset

```diff
@@ -0,0 +1,83 @@
+local assetHelper = asset.require('util/asset_helper')
+local heliosphereTransforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+
+
+local transferFunctions = asset.localResource("transferfunctions")
+local masVelocityColorTable = transferFunctions .. "/velocity_fieldlines.txt"
+local masDensityColorTable = transferFunctions .. "/density_fieldlines.txt"
+
+local fieldlinesDirectory = asset.syncedResource({
+    Name = "Bastille Day MAS Fieldlines",
+    Type = "HttpSynchronization",
+    Identifier = "bastille_day_mas_fieldlines",
+    Version = 2
+})
+
+local sunRadius = 6.957E8
+
+-- Fieldlies from binaries
+local fieldlines = {
+    Identifier = "MAS_MHD_Fieldlines",
+    Parent = heliosphereTransforms.HNMReferenceFrame.Identifier,
+    --wrong Parent = sunTransforms.SunIAU.Identifier,
+    Renderable = {
+        Type = "RenderableFieldlinesSequence",
+        SourceFolder = fieldlinesDirectory,
+        -- FlowEnabled = false,
+        -- LineWidth = 1.0;
+        AlphaBlendlingEnabled = false,
+        InputFileType = "osfls",
+        ColorTablePaths = {
+            masDensityColorTable,
+            masVelocityColorTable,
+        },
+        ColorTableMinMax = {
+            { 0, 1000000 },
+            { 100, 2000 }
+        },
+        LoadAtRuntime = true,
+        ScaleToMeters = 1.0,
+        SecondsBefore = 24*60*60,
+        SecondsAfter = 24*60*60,
+        SimulationModel = "mas",
+        Color = {
+            Uniform = {0, 0.725, 0.75, 1}
+        }
+    },
+    GUI = {
+        Path = "/Solar System/Heliosphere",
+        Name = "MAS Fieldlines"
+    },
+    Transform = {
+        Scale = {
+            Type = "StaticScale",
+            Scale = sunRadius
+        },
+	    Rotation = {
+        --This is a rotation matrix to go from Carrington longitude referens frame to HEEQ180
+        --which is what the renderables paretn transforms.HNMReferenceFrame.Identifier is
+        --refering to. At the referens time, MAS_seq = 0, 2000-07-14T08:33:37.105 the Carrington
+        --longitude was 309.3 degrees.
+        --Difference from HEEQ => 360-309.3=50.7 
+        --(or 0-309.3 = -309.3 However this leads to the same rotation matrix in the end)
+        --Since OpenSpace supports HEEQ180 and not HEEQ, 180 was added or subtracted
+        -- => a1 = -129.3 and a2 = 230.7
+        --Rotation matrix: (cos a, -sin a, 0)(sin a, cos a, 0)(0, 0, 1) leads to the result.
+	    Type = "FixedRotation",
+            XAxis = {-0.63338087262755016203262119192353, -0.77384020972650618518999944537717, 0.0},
```

Comment:
Missing whitespaces in `{ x, y, z }` (see above). Also for YAxis and ZAxis

Also, is this amount of precision really needed for these numbers?